src/taco-dependency-installer/utils/win32/elevatedInstallerLauncher.ps1

```diff
@@ -4,7 +4,7 @@ $pinfo.FileName = "node.exe"
 $pinfo.UseShellExecute = $true
 $pinfo.WindowStyle = "Hidden"
 $pinfo.Verb = "RunAs"
-$pinfo.Arguments = "$($args[0]) $($args[1]) $($args[2]) $($args[3])"
+$pinfo.Arguments = "$args
```

Comment:
I've just learned that in PowerShell, if you put an array inside quotes, it automatically does a join of the array values into a space-separated string. I've changed it to this way so that we don't have to modify the PowerShell script every time we change what is exchanged between the elevated installer and the dependency installer